### PR TITLE
Updating mockserver. Exclusion of dependency from xom.io7m

### PR DESCRIPTION
**Pull request by panossot**
[WFCORE-4850] : https://issues.redhat.com/browse/WFCORE-4850

Updating mockserver. Exclusion of a dependency from xom.io7m .

### CODE REVIEW

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

File: elytron/pom.xml [hunk 1]
```diff
@@ -163,6 +163,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>commons-io</groupId>
```
Comment (jmesnil) [inline elytron/pom.xml]: you mention `xom.io7m` but exclude `commons-io`. what's the relation between the 2?

Comment (panossot) [reply elytron/pom.xml]: There is a compilation error ...

org.apache.commons:commons-collections4:jar:4.2:test
  \- commons-io:commons-io:jar:2.5:test

 does not exist in 5.9.0

Comment (jmesnil) [reply elytron/pom.xml]: Thanks but where does `xom.io7m` come into the picture?

Comment (panossot) [reply elytron/pom.xml]: For example here : https://github.com/wildfly/wildfly-core/blob/master/testsuite/shared/pom.xml#L176

**Comment by jmesnil** [comment]
@panossot thanks

**Pull request merged**